### ISSUE TITLE
V1.0.0  does NOT console.log() to the terminal, only logs in the browser's console

### ISSUE THREAD

**Issue by shardool** [karma-runner/karma#2187]
### Expected behavior : 

console.log('hello world') should log to the terminal when used with the karma-phantomjs-launcher. The previous version V - 0.13.22 logs to the terminal fine

### Actual behavior: 

does NOT console.log() to the terminal. logs only to the browser's console (chrome's console when karma-chrome-launcher is used)

### Enviroment Details

- Karma version (output of `karma --version`): V 1.0.0
- Relevant part of your `karma.config.js` file

### Steps to reproduce the behaviour

1.Karma with Jasmine
2.Chrome and PhatomJS as the browsers
3.Console Logs fine in Chrome, but NOT in the terminal, the previous version of Karma V 0.13.22 works fine
4. And oh !... I'm using Mac

**Comment by shardool** [comment]
thank you very much @dignifiedquire

**Comment by dignifiedquire** [comment]
You are welcome :) Release should go out in the next hours

**Comment by leoselig** [comment]
I'm still having this issue on `v1.1.0` using `karma-phantomjs-launcher`

**Comment by dignifiedquire** [comment]
@leoselig can you try with a different browser?

**Comment by leoselig** [comment]
@dignifiedquire I just tried with Chrome, same behavior

Details:
I have a `console.log(...)` at the beginning of my spec

I ran:
- with only `browsers: ['Chrome']` and `karma@0.13.22` -> **Logged in terminal**
- with only `browsers: ['PhantomJS']` and `karma@0.13.22` -> **Logged in terminal**
- with only `browsers: ['Chrome']` and `karma@1.1.0` -> **Not logged in terminal**
- with only `browsers: ['PhantomJS']` and `karma@1.1.0` -> **Not Logged in terminal**

**Comment by shardool** [comment]
It works for me though. I just tried Karma@1.1.0 and it console logs both, to the terminal and the browser's console. Here is my karma.conf.js. (You may have to explicitly add the launchers to the plugins:[ ] array in the karma.conf file)
```
// Karma configuration
// Generated on Tue Jun 28 2016 23:13:27 GMT-0700 (PDT)
'use strict';
module.exports = function(config) {
    var path = require('path');
    config.set({

    // base path that will be used to resolve all patterns (eg. files, exclude)
    basePath: './',


    // frameworks to use
    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
    frameworks: ['jasmine'],


    // list of files / patterns to load in the browser
    files: [
    'specLoader.js',
    'node_modules/sinon/pkg/sinon.js'
    ],


    // list of files to exclude
    exclude: [
    ],


    // preprocess matching files before serving them to the browser
    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
    preprocessors: {
        'specLoader.js':['webpack']
    },


    // test results reporter to use
    // possible values: 'dots', 'progress'
    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
    reporters: ['progress','coverage'],


    // web server port
    port: 9876,


    // enable / disable colors in the output (reporters and logs)
    colors: true,


    // level of logging
    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
    logLevel: config.LOG_INFO,


    // enable / disable watching file and executing tests whenever any file changes
    autoWatch: true,


    // start these browsers
    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
    browsers: ['Chrome', 'PhantomJS'],


    // Continuous Integration mode
    // if true, Karma captures browsers, runs the tests and exits
    singleRun: false,

    // Concurrency level
    // how many browser should be started simultaneous
    concurrency: Infinity,
    plugins:[
    require('karma-webpack'),
    require('karma-jasmine'),
    require('karma-chrome-launcher'),
    require('karma-phantomjs-launcher'),
    require('karma-coverage')
    ],
    webpack:{
        module:{
            preLoaders:[{
                test:/\.js$/,
                include:path.resolve('sut'),
                loader:'istanbul-instrumenter-loader'
            }]
        }
    }
});
};
```

**Comment by 4kochi** [comment]
Should be resolved by #2220
